cmd/tk/env.go

```diff
@@ -64,6 +64,10 @@ func envSetCmd() *cobra.Command {
 
 		viper.Reset()
 		cfg := setupConfiguration(path)
+		if cfg == nil {
+			fmt.Printf("failed to get config from `%s`\n", path)
+			os.Exit(1)
```

Comment:
I prefer 
```go
if cfg == nil {
  log.Fatalf("Failed to load an environment at `%s`.\nMake sure it exists and is properly configured. See https://tanka.dev/environments/ for details.", path)
}

```

That error message is clearer to read and points the user in the right direction to fix the issue. Furthermore, you an use the `log` package to `Fatalf`, instead of `os.Exit(1)`.